web_external/Front/frontPage.styl

```diff
@@ -1,37 +1,69 @@
 .isic-frontpage
-  $highlightColor = #005ce6
+  position relative
 
-  font-family "Open Sans Condensed", sans-serif
+  background-image url("./lesiondatasets.png")
+  background-size auto 460px
+  background-repeat repeat-x
 
-  .isic-frontpage-header
-    background linear-gradient(to bottom, #fff, #e6f7ff)
-    background-size 100%
-    background-attachment scroll, fixed
-    height 91px
-    letter-spacing 1px
+  &::before
+    // content is required to enable pseudo-elements
+    content ""
+    position absolute
+    height 460px
+    width 100%
+    background linear-gradient(0deg, #000000, #414141)
+    left 0
+    top 0
+    opacity 0.75
 
-    h1
-      font-size 28px
-      font-weight 400
-      color $highlightColor
+  .isic-frontpage-banner
+    text-align center
+    color #ffffff
+    height 340px
 
+    h1
+      margin-top 95px
+      font-size 50px
+      font-weight bold
     h2
-      font-size 14px
-      color #666
-      margin-top 10px
+      font-size 16px
+
+
+  .isic-frontpage-panels
+    background #ffffff
+    opacity 1
+    padding 40px 20px
+    position relative
 
-  .isic-frontpage-body
-    text-align justify
+    .isic-frontpage-panel
```

Comment:
This could have `cursor pointer` to change the mouse cursor to indicate the navigation action.